src/value.c

```diff
@@ -80,8 +80,6 @@ void RSValue_Clear(RSValue *v) {
       // free strings by allocation strategy
       switch (v->strval.stype) {
         case RSString_Malloc:
```

Comment:
yes. I was confused from seeing RSString_Malloc and thought it might be using the wrong allocator until I read through the code and saw it was treated identically to RSString_RMAlloc.